bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/RoomCapability.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.handler.capability;
+
+import static org.openhab.binding.netatmo.internal.NetatmoBindingConstants.*;
+import static org.openhab.binding.netatmo.internal.utils.ChannelTypeUtils.commandToQuantity;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.netatmo.internal.action.RoomActions;
+import org.openhab.binding.netatmo.internal.api.data.NetatmoConstants.MeasureClass;
+import org.openhab.binding.netatmo.internal.api.data.NetatmoConstants.SetpointMode;
+import org.openhab.binding.netatmo.internal.handler.CommonInterface;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.thing.binding.ThingHandlerService;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@link RoomCapability} give the ability to read weather station api
+ *
+ * @author Gaël L'hopital - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class RoomCapability extends Capability {
+    private final Logger logger = LoggerFactory.getLogger(RoomCapability.class);
+
+    public RoomCapability(CommonInterface handler) {
+        super(handler);
+    }
+
+    // Command handling capability
+    @Override
+    public void handleCommand(String channelName, Command command) {
+        if (CHANNEL_SETPOINT_MODE.equals(channelName)) {
+            SetpointMode targetMode = SetpointMode.valueOf(command.toString());
```

Comment:
Please catch an invalid command value leading to no SetpointMode

Reply:
Done